Lib/ctypes/macholib/__init__.py

```diff
@@ -6,4 +6,10 @@
 And also Apple's documentation.
 """
 
-__version__ = '1.0'
+def __getattr__(name):
+    if name == "__version__":
+        from warnings import _deprecated
+
+        _deprecated("__version__", remove=(3, 17))
```

Comment:
PEP 387 says:

> It is preferred, though, to wait 5 years before removal (e.g., warn starting in Python 3.10, removal in 3.15; this happens to coincide with the current lifetime of a minor release of Python).

```suggestion
        _deprecated("__version__", remove=(3, 20))
```


Or is there a reason to rush this?

Reply:
I think it'd be okay to remove after the minimum two years and it'd be nice to remove this clutter, but there's no rush: this PR adds comments telling not to bother changing these lines.